index.html

```diff
@@ -1,80 +1,78 @@
-          </div>
+<body>
+  <header>
+    <nav class="nav-bar">
+      <div class="page--logo">
+        <a class="page--link " href="#">lengushuru</a>
+      </div>
+      <div class="nav--menu" id="nav-toggle">
+        <div></div>
+        <div></div>
+        <div></div>
+      </div>
+      <div class="nav-menu" id="nav-menu">
+        <div class="nav-close" id="nav-close">
+          <i class="bx bx-x"></i>
         </div>
-        <div class="desk-top">
-          <a class="nav-ite" href="#portfolio">Portoflio</a>
-          <a class="nav-ite" href="#about-me">About</a>
-          <a class="nav-ite" href="#container">Contact</a>
+        <div class="nav-links">
+          <a class="nav-item" href="#portfolio">Portoflio</a>
+          <a class="nav-item" href="#about-me">About</a>
+          <a class="nav-item" href="#container">Contact</a>
+          <a class="nav-item" href="#container"><img src="./images/mail_icon.png" alt=""></a>
         </div>
-      </nav>
-    </header>
-    <section class="hero">
+      </div>
+      <div class="desk-top">
+        <a class="nav-ite" href="#portfolio">Portoflio</a>
+        <a class="nav-ite" href="#about-me">About</a>
+        <a class="nav-ite" href="#container">Contact</a>
+        <img src="./images/mail_icon.png" alt="">
+      </div>
+    </nav>
+  </header>
+  <section class="hero">
+    <div class="hero-text">
       <h1 class="main_title">Hey There.<br>
         I’m Charles</h1>
       <h3 class="abt_hero">I’am a Software Developer</h3>
       <p class="description_text">I can help you build a product , feature or website Look through some of my work and
         experience! If you like what you see and have a project you need coded,
         don’t hestiate to contact me.</p>
-      <ul class="socia_icons">
-        <li><a href="#"><img src="images/github.png" alt=""></a></li>
-        <li><a href="#"><img src="images/Linkedin-icon.png" alt=""></a></li>
-        <li><a href="#"><img src="images/peace-icon.png" alt=""></a></li>
-        <li><a href="#"><img src="images/tweeter-icon.png" alt=""></a></li>
-        <li><a href="#"><img src="images/Vector(2).png" alt=""></a></li>
-      </ul>
-    </section>
-    <section id="work-section" class="portfolio">
-      <div class="work-title">
-        <h2>My Resent work</h2>
-        <div></div>
-      </div>
-      <div class="container">
-        <div class="work1">
-          <div class="background"></div>
+    </div>
+
+    <ul class="socia_icons">
+      <li><a href="#"><img src="images/github.png" alt=""></a></li>
+      <li><a href="#"><img src="images/Linkedin-icon.png" alt=""></a></li>
+      <li><a href="#"><img src="images/peace-icon.png" alt=""></a></li>
+      <li><a href="#"><img src="images/tweeter-icon.png" alt=""></a></li>
+      <li><a href="#"><img src="images/Vector(2).png" alt=""></a></li>
+    </ul>
+  </section>
+  <section id="work-section" class="portfolio">
+    <div class="work-title">
+      <h2>My Resent work</h2>
```

Comment:
- [ ] Kindly try to match the design in Figma, the `font-size` and `font-family` here are not the same 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/213730539-afcc0a39-7b10-47c7-8ce8-e141a683b2b2.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/213730589-8ca327a2-9a1e-4a69-8356-39fe783c3876.png)